src/js/descript.js

```diff
@@ -135,5 +167,41 @@
         return containerName ? this._containers[containerName] : this._containers;
     };
 
+    /**
+     * Replaces string patterns in inline scripts. Patterns can be
+     * @param containerName
```

Comment:
No, this just replaces text within the inline script. It doesn't move or remove scripts from any of the containers.